### PR TITLE
add premium menu and new arrivals section and some minor style fixes

### PR DESCRIPTION
**Pull request by Saminsfw**

### CODE REVIEW

**Comment by UmmarHamid** [comment]
@Saminsfw I cannot find any screenshot attached, hence review is on hold!

**Comment by Saminsfw** [comment]
menu

**Comment by Saminsfw** [comment]
menu2

File: index.html [hunk 1]
```diff
@@ -91,5 +91,77 @@
         <img src="/images/pizza.svg" alt="Pizza Slice" />
       </div>
     </div>
+    <!-- Product Menu -->
```
Comment (UmmarHamid) [inline index.html:94]: Nice to have these section comments, it makes reading code easier!

File: styles.css [hunk 2]
```diff
@@ -11,7 +11,7 @@ header {
   justify-content: space-around;
   align-items: center;
   height: 80px;
-  padding: 0 185px 0 147px;
+  /* padding: 0 185px 0 147px; */
```
Comment (UmmarHamid) [inline styles.css]: Lets remove this

File: styles.css [hunk 3]
```diff
@@ -33,7 +33,7 @@ header {
 .hero-section {
   display: flex;
   justify-content: center;
-  padding: 0 209px 0 159px;
+  /* padding: 0 209px 0 159px; */
```
Comment (UmmarHamid) [inline styles.css]: same

File: styles.css [hunk 4]
```diff
@@ -11,7 +11,7 @@ header {
   justify-content: space-around;
   align-items: center;
   height: 80px;
-  padding: 0 185px 0 147px;
+  /* padding: 0 185px 0 147px; */
   margin-bottom: 61px;
```
Comment (UmmarHamid) [inline styles.css]: lets round this of to multiple of 4

File: styles.css [hunk 5]
```diff
@@ -67,7 +67,8 @@ header {
   color: #dad8d7;
 }
 .hero-button,
-.about-button {
+.about-button,
+.discount-button {
   padding: 17px 28px;
```
Comment (UmmarHamid) [inline styles.css]: lets normalize

**Review by UmmarHamid — CHANGES_REQUESTED** [review]

**Review by UmmarHamid — APPROVED** [review]

**Pull request merged**